docs/components/index.md

```diff
@@ -0,0 +1,360 @@
+---
+title: Blazor components
+author: guardrex
+description: Learn how to create and use Blazor components, the fundamental building blocks of Blazor apps provided by compiled Razor or C# files.
+manager: wpickett
+ms.author: riande
+ms.custom: mvc
+ms.date: 04/14/2018
+ms.prod: asp.net-core
+ms.technology: aspnet
+ms.topic: article
+uid: client-side/blazor/components/index
+---
+# Blazor components
+
+By [Luke Latham](https://github.com/guardrex)
+
+[!INCLUDE[](~/includes/blazor-preview-notice.md)]
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnet/Blazor.Docs/docs/components/common/samples/)
+
+To download the sample:
+
+1. Download and unzip the [Blazor.Docs repository zip file](https://codeload.github.com/aspnet/Blazor.Docs/zip/master).
+1. Use the URL in the [sample link](https://github.com/aspnet/Docs/tree/master/aspnet/Blazor.Docs/docs/components/common/samples/) to navigate to the sample folder.
+1. See the [Get started](xref:client-side/blazor/get-started) topic for prerequisites.
```

Comment:
Could we put the "prerequisites" item first?

The phrasing of the second item is extremely strange. I don't know how to interpret what it's telling people to do. What sample link - the actual link whose text is "sample link", and if so, what is one meant to do with that URL? Could this be rephrased for clarity?

Reply:
Perhaps so ... but there are larger forces at work here. Later, we must use this language on pain-of-death ☠️ :smile:. This is the text we use on the aspnet/Docs side. https://github.com/aspnet/Docs/blame/master/aspnetcore/tutorials/index.md#L84

I wrote detailed instructions on the .NET Core docs side on https://github.com/dotnet/docs/pull/1832 ... that are live at https://docs.microsoft.com/dotnet/samples-and-tutorials/#viewing-and-downloading-samples.

Because we're in a separate repo right now, yes, I'll institute the detailed instructions elsewhere (docs index page or tutorials index page) along the lines of how the .NET Core docs handle it and link it via a "how to download" link here. When this content moves into the aspnet/Docs, we'll have to drop it tho and go with the direction that aspnet/Docs takes.